pkgs/by-name/kl/klocalizer/package.nix

```
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, python3Packages
+}:
+
+let
+  version = "4.5.1";
+in
+python3Packages.buildPythonPackage {
+  pname = "klocalizer";
+  inherit version;
+
+  src = fetchFromGitHub {
+    owner = "paulgazz";
+    repo = "kmax";
+    inherit version;
+    rev = "v${version}";
+    hash = "sha256-fNs/i8kjc3P6XvX5lvSvNLfhoetr0plNVG+KcRD/Otc=";
+  };
+
+  # complains cryptically about wanting `--use-pep517`
+  doCheck = false;
+
+  propagatedBuildInputs = with python3Packages; [
+    packaging
+    z3
+    setuptools
+    regex
+    whatthepatch
+    requests
+  ];
+
+  meta = {
+    description = "Linux Kconfig constraint SAT/SMT solver";
+    license = [ lib.licenses.gpl2 ];
```

```suggestion
    license = [ lib.licenses.gpl2Plus ];
```

https://github.com/paulgazz/kmax/blob/d0fe92db13ca97140385cd39977d937d1ee92d4c/GPL-2.0#L4